odata-url-conventions/Appendix.md

```diff
@@ -50,6 +50,10 @@ https://www.rfc-editor.org/info/rfc3986.
 _Leiba, B., "Ambiguity of Uppercase vs Lowercase in RFC 2119 Key Words", BCP 14, RFC 8174, DOI 10.17487/RFC8174, May 2017_.
 https://www.rfc-editor.org/info/rfc8174.
 
+###### <a name="_url">[URL]</a>
+_URL Living Standard._  
+https://url.spec.whatwg.org/.
```

Comment:
Cross-check with OASIS citation rules for W3C specs

Reply:
https://www.w3.org/TR/html/ redirects to https://html.spec.whatwg.org/multipage/ and this contains no citation instructions.